event-based-simulators/main/scripts/README.md

```diff
@@ -0,0 +1,32 @@
+# Generic scripts information
+This scripts folder contains:
+- The **ExportOrImportProfileData.py** which is used to export measurements or alarms data from devices
+- The **Environment.py** which is used to setup environment parameters
+
+## Setup environment parameters:
+Input base url, tenant ID, username, password in Environment.py file
+
+## Install cumulocity-python-api package
+Follow the instructions in: https://github.com/SoftwareAG/cumulocity-python-api
+
+### Installation from PyPI
+```shell
+pip install c8y_api
+```
+
+## Run the script
+If the environment parameters were not setup, they can be input as arguments when running the script
+```shell
+python ExportProfilesData.py [-h] [--baseurl BASEURL] [--id-tenant ID_TENANT]
+                             [--username USERNAME --password PASSWORD]
+                             [--data-type {measurements,alarms}]
+                             [--mode {all,specific}]
+```
+optional arguments:\
+  -h, --help : show this help message and exit \
+  --baseurl BASEURL, -b BASEURL : Base URL of the tenant\
+  --id-tenant ID_TENANT, -i ID_TENANT : Tenant ID \
```

Comment:
- the typical name for it is "tenantId"

Reply:
ok, I would like to use -i for this argument so I used id-tenant to match that idea. I will use -tenantId and -t for the consistency